src/charts/legend.js

```diff
@@ -502,6 +503,21 @@ define(function(require){
             return this;
         };
 
+        /**
+         * Gets or Sets the unit of the value
+         * @param  {boolean} _x Desired unit
+         * @return {boolean | module} Current unit or Legend module to chain calls
+         * @public
+         */
+        exports.unit = function(_x) {
```

Comment:
We would need to add a test for this getter/setter.

We would probably benefit to have another test checking that the unit is actually being added when rendering a legend with unit.